docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -136,6 +136,9 @@ If the Application Root is exposed in a different path and needs to be redirecte
 The annotation `nginx.ingress.kubernetes.io/affinity` enables and sets the affinity type in all Upstreams of an Ingress. This way, a request will always be directed to the same upstream server.
 The only affinity type available for NGINX is `cookie`.
 
+!!! attention
+    If more than one Ingress is defined for a host and at least one Ingress uses `nginx.ingress.kubernetes.io/affinity: cookie`, then only paths on the Ingress using `nginx.ingress.kubernetes.io/affinity` will use session cookie affinity. All paths defined on other Ingresses for the host will be load balanced based on [consistent hashing](http://nginx.org/en/docs/http/ngx_http_upstream_module.html#hash). 
+
```

Comment:
> All paths defined on other Ingresses for the host will be load balanced based on [consistent hashing]

That's not correct. It will pick backend server randomly in this case. It's a very hacky solution at the moment: we generate a key for `chash` randomly, and therefore whatever `chash` returns will be random as well.

> http://nginx.org/en/docs/http/ngx_http_upstream_module.html#hash

we don't use this, we use chash implementation from https://github.com/openresty/lua-resty-balancer